packages/httpsnippet-client-api/package.json

```
@@ -9,7 +9,7 @@
     "lint:types": "tsc --noEmit",
     "prebuild": "rm -rf dist/",
     "prepack": "npm run build",
-    "test": "jest --coverage"
+    "test": "vitest run --coverage"
```

nit but [per their docs](https://vitest.dev/guide/cli.html#vitest), `vitest` auto-runs in watch mode in dev and non-watch in CI. thoughts on this?

```suggestion
    "test": "vitest --coverage"
```